website/docs/r/compute_region_instance_group_manager.html.markdown

```diff
@@ -34,9 +34,10 @@ resource "google_compute_health_check" "autohealing" {
 resource "google_compute_region_instance_group_manager" "appserver" {
   name = "appserver-igm"
 
-  base_instance_name = "app"
-  instance_template  = "${google_compute_instance_template.appserver.self_link}"
-  region             = "us-central1"
+  base_instance_name  = "app"
+  instance_template   = "${google_compute_instance_template.appserver.self_link}"
+  region              = "us-central1"
+  distribution_policy = ["zones/us-central1-a", "zones/us-central1-b", "zones/us-central1-f"]
```

Comment:
do we need "zones/" prefix?

Reply:
We do.

The API requires you to pass it a "URL" of the zone.  Without the prefix, it doesn't work, but with just the `zones/` prefix, it properly extrapolates it out to something like `https://www.googleapis.com/compute/beta/projects/PROJECT_ID/zones/europe-west3-c`.

I'm still looking into the code to see if there's an existing precedent/piece of code that could properly generate/parse the full URL from just the zone name.

Reply:
I had exactly same question.. I think it would be better if we deal with this in function if we have to, as opposed to expecting people to provide it. It is confusing, since everywhere else it's just a zone name

Reply:
Yeah, I think I bake have it add this for the user, but I just need to make sure that there's no special reason that the API requires a special format instead of just `zone` by itself, as the code would break those other use cases.